### ISSUE TITLE
"column out of bounds" on minitwit

### ISSUE THREAD

**Issue by vmarkovtsev** [bblfsh/python-driver#102]
Merge https://github.com/bblfsh/client-python/pull/38 to print `errors`

Then
```
git clone https://github.com/pallets/flask
python3 -m bblfsh -f flask/examples/minitwit/minitwit/minitwit.py >/dev/null
```

And you get an error from bblfsh:
```
column out of bounds: 63 [1, 51]
```
The file is parsed though.

**Comment by juanjux** [comment]
Yes, that's really the SDK indicating that it could not calculate the offset from the provided line and column (because in this case the column is 63, but the real indicated line only goes to 51). It doesn't stop parsing as you saw but will leave the offset of that node to 0 in the UAST.

I'll take a look.

**Comment by juanjux** [comment]
Found the problem. Multi-line nodes with tokens were not having their starting line number fixed from the native token (like it's done for the starting column and ending line and column).

Fixed by #111.
